internal/command/status/machines.go

```diff
@@ -169,12 +169,18 @@ func RenderMachineStatus(ctx context.Context, app *api.AppCompact, out io.Writer
 			if len(machine.Config.Standbys) > 0 {
 				hasStandbys = true
 			}
+			var role = "N/A"
```

Comment:
can it be an empty string? It looks too heavy to show N/A when generally not used.